src/bptl/work_units/kownsl/tasks.py

```diff
@@ -20,6 +22,27 @@ def get_client(task: BaseTask, alias: str = "kownsl") -> ZGWClient:
     return client
 
 
+def get_review_request(task: BaseTask) -> dict:
+    """
+    Get a single review request from kownsl.
+    """
+    variables = task.get_variables()
+
+    zaak_url = check_variable(variables, "hoofdZaakUrl")
```

Comment:
Since this is in the accordering sub process the zaakUrl is not available.

Reply:
Mmm, I see that approval status is also using zaakUrl. We're using the variables for the sub process to map zaakUrl to hoofdZaakUrl though in both the accordering and advisering sub processes. 

Align everything...?

Reply:
the BPTL work units should not assume anything about sub-process, for all they are aware, they run in the main process.

I prefer consistency in the work units, so `zaakUrl` everywhere. If this is being called in the context of a sub-process, it's up to the process modeller to map `hoofdZaakUrl` to `zaakUrl` for this particular service task. In Camunda, this can be done using the input/output parameters.